dynamo/plot/__init__.py

```diff
@@ -63,6 +63,7 @@
     arcPlot,
     circosPlot,
     hivePlot,
+    circos_plot,
```

Comment:
what is the difference of circos_plot andcircosPlot? 

and can you run both plotting functions?

Reply:
CircosPlot cannot be run with nxviz now. circos_plot is compatible with the latest nxviz package.
I leave CircosPlot there for backward compatibility, in case users have old nxviz package installed. And new code will not change their notebooks' behavior with old nxviz package. Please let me know if you would like to remove it.

Reply:
please then  deprecated the previous code (changed it ot circosPlot_deprecated) and replace circosPlot with the new code. I don't like the idea to have a new function `circos_plot`. I mean we only have the circosPlot